go/enclave/components/sigverifier.go

```diff
@@ -0,0 +1,45 @@
+package components
```

Comment:
I'm not sure, but it kind of holds state and needs access to the storage because of the `FetchAttestedKey`. Was thinking it can be extended further to remove the process network secrets stuff from the enclave.